src/modules/map/components/toolbox/TimeSliderButton.vue

```diff
@@ -54,7 +54,9 @@ function toggleTimeSlider() {
                 'dev-disclaimer-present': hasDevSiteWarning,
             }"
         >
-            <TimeSlider v-if="isTimeSliderActive" />
+            <div class="d-flex justify-content-center">
+                <TimeSlider v-if="isTimeSliderActive" />
+            </div>
```

Comment:
the dropdown is not displayed correclty (year is cut off) on small device 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/c2d00c65-802b-4a7d-b3ee-19caaf7f59cd)